packages/api/schema/input.json

```diff
@@ -13,40 +73,68 @@
       ],
       "type": "string"
     },
-    "InterventionPeriod": {
+    "Intervention": {
       "additionalProperties": false,
       "properties": {
-        "caseIsolation": {
-          "$ref": "#/definitions/Intensity",
-          "description": "The level to which individuals with symptoms self-isolate."
+        "intensity": {
+          "$ref": "#/definitions/Intensity"
         },
         "reductionPopulationContact": {
```

Comment:
A little confused how this will work if the `reductionPopulationContact` is associated with an intervention, rather than an `interventionPeriod`. Are we expecting that everything is additive?

Reply:
Or...I think this should just be removed...